introduction.md

```diff
@@ -0,0 +1,5 @@
+Lorem ipsum dolor sit amet, consectetur magna aliqua. Ut enim ad minim veniam,
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.